### DISCUSSION TITLE
GitHub Actions dependency caching

### DISCUSSION THREAD

**Discussion by medvednikov** [vlang/v#11588] [Ideas]
https://stackoverflow.com/questions/47603077/how-to-cache-packages-which-have-been-installed-via-apt-get-in-travis-ci

Azure still don't have a way to cache dependencies. Once they do, it should be used.

**Comment by alexesprit** [comment]
@medvednikov is this issue still relevant?

**Comment by medvednikov** [comment]
Yes, but with GitHub actions.
